module/solutions/helper.js

```diff
@@ -2354,7 +2354,7 @@ module.exports = class SolutionsHelper {
 	 * @returns {Object}
 	 */
 
-	static assignedProjects(userId, pageSize, pageNo, search, filter) {
```

Comment:
@Prajwal17Tunerlabs We can keep the pageSize and pageNo , when calling the function pass it as ' '.

Reply:
ok, followed as said above